code/modules/assembly/igniter.dm

```diff
@@ -24,6 +24,9 @@
 	var/turf/location = get_turf(loc)
 	if(location)
 		location.hotspot_expose(1000,1000)
+		for(var/T in location.contents)
+			var/atom/A = T
+			A.fire_act()

```

Comment:
You can rig an igniter to a remote, put it inside any container, give it to someone and then light them on fire from a distance.

This is just not well thought out.